test/umzug.test.ts

```diff
@@ -93,6 +93,27 @@ describe('custom context', () => {
 		expect(spy).toHaveBeenCalledTimes(1);
 	});
 
+	test(`create with custom template extension doesn't cause bogus warning`, async () => {
+		const syncer = fsSyncer(path.join(__dirname, 'generated/create-custom-template'), {});
+		syncer.sync();
+
+		const umzug = new Umzug({
+			migrations: {
+				glob: ['*.js', { cwd: syncer.baseDir }],
+			},
+			logger: undefined,
+			create: {
+				folder: syncer.baseDir,
+				template: filepath => [[`${filepath}.x.js`, `/* custom template */`]],
+			},
+		});
+
+		await umzug.create({ name: 'test' });
```

Comment:
Should have pushed the test in a separate commit to prove this but: this was reproducing the bug described in #559 until the change to `src/umzug.ts` above.